Formula/spirv-tools.rb

```diff
@@ -40,7 +40,9 @@ def install
 
     mkdir "build" do
       system "cmake", "..", *std_cmake_args,
+                            "-DBUILD_SHARED_LIBS=ON",
                             "-DSPIRV_SKIP_TESTS=ON",
+                            "-DSPIRV_TOOLS_BUILD_STATIC=OFF",
```

Comment:
This looks like a pretty big change to the build. We should probably bump the revision here.

Reply:
Good call, I will revision bump.